dockerfile/anaconda-rpm/requirements.txt

```diff
@@ -0,0 +1,7 @@
+# pip package install list
+# storing it as an independent file enables Dependabot to bump versions with noise and attention
+# silent updates of pylint and astroid usually cause breakages
+# for format details see https://pip.pypa.io/en/stable/reference/requirements-file-format/
+
+# unit tests of all kinds
+pocketlint  # translatable strings and translations, used by pylint and glade tests
```

Comment:
Please pin the version of pylint. I don't think it's necessary for pocketlint but pylint is getting broken quite often.